ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateProvider.fs

```diff
@@ -256,11 +255,15 @@ type FSharpOverridingMembersBuilder() =
             for node in range do
                 if not (isFirstMeaningfulNodeOnLine node) then () else
 
-                let diff = node.Indent - (desiredIndent + additionalIndent)
+                let diff =
+                    if node = typeRepr then
```

Comment:
Please use `==`, we're interested in reference equality only when working with these tree nodes.